src/main/drivers/dshot_bitbang.c

```diff
@@ -475,8 +476,12 @@ static void bbWriteInt(uint8_t motorIndex, uint16_t value)
         return;
     }
 
-    // If there is a command ready to go overwrite the value and send that instead
+    // fetch requestTelemetry from motors. Needs to be refactored.
+    motorDmaOutput_t * const motor = getMotorDmaOutput(motorIndex);
+    bbmotor->protocolControl.requestTelemetry = motor->protocolControl.requestTelemetry;
```

Comment:
Wouldn't it be just as easy, and cleaner, to set `requestTelemetry` in `src/main/sensors/esc_sensor.c` where `requestTelemetry` for non-bitbanged Dshot is set?

Reply:
I’m not sure. Really this should be set at a data structure shared between bbshot and timer dshot. Since it’s a hack anyway which should be fixed once 4.1 is done it doesn’t matter much where it lives in my view.

Reply:
Fair enough - I guess with bitbanged Dshot we will have to consider another refactor of the Dshot code after 4.1 has gone out.